datacommons/datacommons.py

```diff
@@ -452,6 +454,56 @@ def get_states(self, country, new_col_name, max_rows=100):
 
     return pd.concat([type_row, dcid_column], ignore_index=True)
 
+
+  def get_places_in(self, place_type, container_dcid, col_name, max_rows=100):
+    """Get a list of places that are contained in a higher level geo places.
+
+    Args:
+      place_type: The place type, like "City".
+      container_dcid: The dcid of the container place.
+      col_name: Column name for the returned state column.
+      max_rows: max number of returend results.
+
+    Returns:
+      A pandas.DataFrame with dcids of the contained place.
+    """
+    assert self._inited, 'Initialization was unsuccessful, cannot execute Query'
+    assert place_type in _PLACES, 'Input place types are not supported'
+
+    # Get the type of the container place.
+    type_query = 'SELECT ?type, dcid ?node {dcid}, subType ?node ?type'.format(
+        dcid=container_dcid)
+    query_result = self.query(type_query)
+    assert query_result['type'].count() == 1, (
+        'Type of the container dcid not found')
+    container_type = query_result['type'][0]
+
+    # Sanity check the type information.
+    place_type_ind = _PLACES.index(place_type)
+    container_type_ind = _PLACES.index(container_type)
+    assert container_type_ind > place_type_ind, (
+        'Asked place should be of lower level type of the container')
```

Comment:
nit: 'Requested place should be of a lower level type than the container'

Reply:
Updated